examples/fluent-bit/filter-multiline/README.md

```diff
@@ -0,0 +1,51 @@
+### FireLens Example: Concatenate Multiline or Stack trace log messages
+
+Multiline Filter is availble on `aws-for-fluent-bit` >= v2.18.0. It helps to concatenate messages that originally belong to one context but were split across multiple records or log lines. Common examples are stack traces or applications that print logs in multiple lines. More information could be found in [Fluent Bit official doc](https://docs.fluentbit.io/manual/pipeline/filters/multiline-stacktrace).
+
+With this filter, you are able to use Fluent Bit built-in parses with auto detection and multi format support on:
+- go
+- python
+- ruby
+- java (Google Cloud Platform Java stacktrace format)
```

Comment:
nit I think let's avoid mentioning our competitors in an AWS example. Is Google format useful for AWS customers?

Reply:
will delete: `(Google Cloud Platform Java stacktrace format)`